app/views/display_tasks/tasks.html.erb

```diff
@@ -1,37 +1,29 @@
 <h1>TASK LIST</h1>
 
+<%= button_to "Add new task", "/tasks/new", method: "get" %>
+
 <table>
   <tr>
-    <th> Task # </th>
     <th> Name </th>
     <th> Description </th>
+    <th> Assigned to </th>
     <th> Completed? </th>
-    <th> Date Completed </th>
   </tr>
 
-  <% @tasks.each_with_index do |task, index| %>
+  <% @tasks.each do |task| %>
     <div>
       <tr>
-        <td> <%= task[:id] %> </td>
-        <td> <%= link_to "#{task[:name]}", "/task/#{task[:id]}" %> </td>
-        <td> <%= task[:description] %> </td>
-
-        <% if task[:completed_at] == nil %>
-          <td>No</td>
-        <% else %>
-          <td>Yes</td>
-        <% end %>
-
-        <td> <%= task[:completed_at] %> </td>
+        <td> <%= link_to "#{task.name}", "/tasks/#{task.id}" %> </td>
+        <td> <%= task.description %> </td>
+        <td> <%= Task.find(task.id).person.name %> </td>
```

Comment:
The `task` variable stores the task object so you shouldn't retrieve it again here